deploy/src/amb_erc677_to_erc677/foreign.js

```diff
@@ -2,9 +2,16 @@ const Web3Utils = require('web3-utils')
 const { web3Foreign, FOREIGN_RPC_URL } = require('../web3')
 const { deployContract, privateKeyToAddress, upgradeProxy } = require('../deploymentUtils')
 const {
-  foreignContracts: { EternalStorageProxy, ForeignAMBErc677ToErc677: ForeignBridge }
+  foreignContracts: {
+    EternalStorageProxy,
+    ForeignAMBErc677ToErc677: ForeignBridge,
```

Comment:
Does it make sense to rename `ForeignBridge` to `ForeignBridgeAbsoluteDailyLimit`?